austation/code/modules/mining/equipment/marker_beacons.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/stack/marker_beacon/attack_self(mob/user)
+	if(GLOB.total_beacons >= GLOB.max_beacons) //check that  there are not 100 in the world already
+		to_chat(user, "<span class='warning'>You can not place another beacon!</span>")
```

Comment:
Just give this a more explanative reason and it should be good to go!

Reply:
Terra wants the limit removed